### DISCUSSION TITLE
Feature Request: ODBC/JDBC datasource (eg. for Databricks Spark Cluster)

### DISCUSSION THREAD

**Discussion by stepanurban** [grafana/grafana#48705] [Data Sources]
Would it be possible to add a generic ODBC/JDBC datasource? It would allow to connect various different databases including Databricks Spark SQL endpoints.

**Comment by stepanurban** [comment]
What would be the best way to do it? Can somebody guide me? I need to query Spark SQL endpoint. Is it possible to simply modify some existing datasource?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
